lib/grpcServer/handlers/platform/broadcastStateTransitionHandlerFactory.js

```diff
@@ -53,11 +51,7 @@ function broadcastStateTransitionHandlerFactory(rpcClient, handleAbciResponseErr
     }
 
     if (result.code !== undefined && result.code !== 0) {
```

Comment:
If code is undefined it must throw an internal error. I would pass a response to the create function in all cases if `code !== 0`